presto-main/src/main/java/com/facebook/presto/sql/planner/sanity/ValidateDependenciesChecker.java

```diff
@@ -426,8 +427,7 @@ public Void visitIndexSource(IndexSourceNode node, Set<Symbol> boundSymbols)
         @Override
         public Void visitTableScan(TableScanNode node, Set<Symbol> boundSymbols)
         {
-            checkArgument(node.getAssignments().keySet().containsAll(node.getOutputSymbols()), "Assignments must contain mappings for output symbols");
-
+            //We don't have to do a check here as TableScanNode has no dependencies.
```

Comment:
This should go in  separate commit. Also please also put a space between`//` and `W`